openlibrary/core/lending.py

```diff
@@ -213,8 +211,8 @@ def get_random_available_ia_edition():
                "+AND+loans__status__status:AVAILABLE"
                "&fl=identifier,openlibrary_edition,loans__status__status"
                "&output=json&rows=1&sort[]=random" % (config_bookreader_host))
-        content = urllib.request.urlopen(url=url, timeout=config_http_request_timeout).read()
-        items = simplejson.loads(content).get('response', {}).get('docs', [])
+        response = requests.get(url, timeout=config_http_request_timeout)
+        items = response.json().get('response', {}).get('docs', [])
```

Comment:
https://staging.openlibrary.org/random worked as expected and results were always available to borrow.